aries_vcx/tests/test_mysql_wallet.rs

```diff
@@ -0,0 +1,102 @@
+#[macro_use]
+extern crate log;
+extern crate serde;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use]
+extern crate serde_json;
+#[cfg(feature = "mysql_test")]
+#[macro_use]
+extern crate sqlx;
+
+#[cfg(feature = "mysql_test")]
+mod dbtests {
+    use sqlx::{Connection, MySqlConnection};
+    use sqlx::Executor;
+    use sqlx::prelude;
+    use sqlx::prelude::*;
+
+    #[cfg(feature = "mysql_test")]
```

Comment:
This feature is configured in the outer module, so I would assume it is not necessary here (applies to the `test` module below as well)